contracts/nft/erc721m/extensions/BucketAuction.sol

```diff
@@ -30,9 +28,10 @@ contract BucketAuction is IBucketAuction, ERC721M {
         uint256 maxMintableSupply,
         uint256 globalWalletLimit,
         address cosigner,
+        uint256 timestampExpirySeconds,
         uint256 minimumContributionInWei,
-        uint64 startTimeUnixSeconds,
-        uint64 endTimeUnixSeconds,
+        uint256 startTimeUnixSeconds,
+        uint256 endTimeUnixSeconds,
```

Comment:
Can we save gas by staying lower on these? Was `uint64` future proof enough?

Reply:
yeah we had uint64 originally, but i purposely converted them to 256 because they are being compared against block.timestamp which is uint256. we were having to convert them anyway